modules/ossm-support-for-istioctl.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-istioctl-tool
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-support-for-istioctl_{context}"]
+= Support for Istioctl
+
+{SMProduct} 3 provides support for a selection of Istioctl commands.
+
+.Supported Istioctl commands
+[cols="1,1"]
+|===
+|Command |Description 
+
+|`admin` | Manage control plane (`istiod`) configuration 
+|`analyze` | Analyze Istio configuration and print validation messages 
+|`completion` | Generate the autocompletion script for the specified shell
+|`create-remote-secret` | Create a secret with credentials to allow Istio to access remote Kubernetes `apiservers` 
+|`help` | Help about any command 
+|`proxy-config`, `pc` | Retrieve information about proxy configuration from Envoy (Kubernetes only) 
+|`proxy-status`, `ps` | Retrieves the synchronization status of each Envoy in the mesh
+|`remote-clusters` | Lists the remote clusters each `istiod` instance is connected to 
+|`validate`, `v` | Validate Istio policy and rules files 
+|`version` | Prints out build version information 
+|`waypoint` | Manage waypoint configuration 
+|`ztunnel-config` | Update or retrieve current Ztunnel configuration.
```

Comment:
The verb tense in here is mixed (Manage, Analyze, Retrieves, Lists, Update) or missing ("Help about any command"). Either verb use is fine here, but make them all the same for parallel presentation :)

Similarly, some phrases have articles (Generate the autocompletion script, Create a secret), some don't (Manage control plane (`istiod`) configuration, Analyze Istio configuration). Probably they all should, so I would make that consistent as well.

Reply:
done